qe_metrics/libs/database.py

```diff
@@ -127,3 +134,44 @@ class JiraIssues(DB_CONNECTION.Entity):  # type: ignore
         customer_escaped = orm.Required(bool)
         date_created = orm.Required(date)
         last_updated = orm.Required(date)
+        date_record_modified = orm.Required(datetime, default=datetime.now)
+
+        @classmethod
+        def create_update_issues(
+            cls, issues: List[Issue], product: "Database.Products", severity: str, jira_server: str
+        ) -> None:
+            """
+            Create or update JiraIssues items in the database from a list of Jira issues.
+
+            Args:
+                issues (List[Issue]): A list of Jira issues
+                product (Database.Products): A product object
+                severity (str): Severity of the issues
+                jira_server (str): Jira server URL
+
+            Returns:
+                List["Database.JiraIssues"]: A list of JiraIssues objects
+            """
+            issues = [_issue for _issue in issues if _issue.fields.issuetype.name.lower() == "bug"]
+            for issue in issues:
+                existing_issue = cls.get(issue_key=issue.key, product=product)
+                if existing_issue:
+                    update_existing_issue(existing_issue, issue, severity)
+                else:
+                    cls(
+                        product=product,
+                        issue_key=issue.key,
+                        title=issue.fields.summary,
+                        url=f"{jira_server}/browse/{issue.key}",
+                        project=issue.fields.project.key,
+                        severity=severity,
+                        status=issue.fields.status.name,
+                        customer_escaped=check_customer_escaped(issue=issue),
+                        date_created=format_issue_date(issue.fields.created),
+                        last_updated=format_issue_date(issue.fields.updated),
+                    )
+
+            delete_closed_issues(
```

Comment:
Do we want to delete closed issues or leave it there and it will be deleted in the cleanup?

Reply:
moreover, i guess products would like to see these bugs (e.g a blocker would probably be addressed rather quickly but you still want this is the stats)

Reply:
@rnetser @myakove sure, that makes sense. I will have to replace this with a function that sets the `status` to closed. The way the JQL queries are written, they will not find closed issues. So we will need a function to find issues in our db that are no longer being returned by the JQL queries (essentially what this does).

@rujutashinde just an FYI, you'll need to account for closed issues being in the db at least for 90 days. I will follow up after I fix the code.

Reply:
@myakove @rnetser I have discussed this with @rujutashinde, and we have decided to mark the "status" of these issues as `stale` in this instance, rather than deleting them.